### PR TITLE
Consistent file naming in client

### PR DESCRIPTION
**Pull request by IanFelton**
Getting consistency around file naming.

this-is-a.controller.js
this-is-a.controller.spec.js
this-is-a.service.js
this-is-a.service.spec

### CODE REVIEW

**Comment by IanFelton** [comment]
@MarchingMountains/prime-academy So far just in the client, but I've put a consistent file naming convention. If everyone uses the same file naming, it's much clearer what files do what and less likely to make mistakes when including a file because you don't have to question whether it's file where someone named it myService.js or my-service.js.

For this project, we will do the following
my.service.js
my.service.spec.js
my-longer-named.controller.js
my-longer-named.controller.spec.js

So basically, no uppercase at all. User hypens to separate longer names.
Use a "." before the type of file and any other extensions like ".spec" 
So:
.controller.js
.service.js
.filter.js
.directive.js
.filter.spec.js
.controller.spec.js

.

**Comment by Katirene** [comment]
Sounds good. Would it make sense for me to compile this in a style guide txt doc for the repo?

**Comment by IanFelton** [comment]
Sounds great

Ian Felton
